classes/opponentPhaseWeakspot.lua

```diff
@@ -95,21 +95,22 @@ function opponentPhaseWeakspot:update(dt)
 		self.y = ARENA_HEIGHT - self.radius
 	end
 
-	self.xDamage = self.x - math.cos(self.angle) * 0.5 * self.radius
-	self.yDamage = self.y - math.sin(self.angle) * 0.5 * self.radius
+	self.xDamage = self.x - math.cos(self.angle) * 0.75 * self.radius
+	self.yDamage = self.y - math.sin(self.angle) * 0.75 * self.radius
```

Comment:
I pushed the "weakspot" a bit farther away from the center so that it's more clear you need to hit it from the back, it looks like it sticks out more